engine/engine.go

```diff
@@ -197,8 +196,17 @@ func (e *engine) registerWebhookTrigger(t *webhookt) {
 	// Add engine instance to inside of trigger, required for starting workflows
 	t.engine = e
 
-	// Let the engine know what urlSuffix are associated with this particular instance of trigger
-	e.triggers["webhook"][t.urlSuffix] = t
+	// Register routes on webhook listener http server
+	e.WebhookListener.RegisterRoute(
```

Comment:
Should we handle the error here? If the route already exists the error will be "swallowed" which will probably be hard to debug.

Reply:
Handled now!